src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -69,6 +69,7 @@ public function testCsrfProtectionForFormsEnablesCsrfProtectionAutomatically()
         $this->assertTrue($container->hasDefinition('security.csrf.token_manager'));
     }
 
+    /** @group legacy */
```

Comment:
Does ``hasDefinition`` trigger a deprecation warning ? This looks weird to me. We should trigger it only when actually getting the service, not when checking whether we have a definition for it.